lib/google_crawler/google/api_client.ex

```diff
@@ -0,0 +1,22 @@
+defmodule GoogleCrawler.Google.ApiClientBehaviour do
+  @callback search(keyword :: String.t()) :: {:ok, String.t()} | {:error, String.t()}
+end
```

Comment:
It's actually a good idea to use a behaviour so make sure that both the actual and mock modules have the same implementation.